src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -56,96 +48,68 @@ function RegisterForm(props) {
   };
 
   const onSubmit = ({ email, password }) => {
-    signInSignUp(email, password, SIGN_UP_URL, fn);
+    signInSignUp(email, password, SIGN_UP_URL, changeStateFn);
   };
+
   return (
     <Box className={styles.registerForm__wrapper} component="form" onSubmit={handleSubmit(onSubmit)}>
```

Comment:
You could consider adding `noValidate` to this form and the login form to prevent default validation done by browser.